scheduler/remote_server.py

```diff
@@ -67,7 +67,7 @@ def plot_timeline(timeline_dict, scd_dir, now):
     """
     fig, ax = plt.subplots()
 
-    event_labels = []
+    event_list = []
```

Comment:
I woudl move this down to line 143 to right before its used.

Reply:
Changed this